src/index.css

```diff
@@ -1,6 +1,7 @@
 #root {
   width: 100%;
   height: 100%;
+  margin: 10px;
```

Comment:
- `.display-container` should have a height of `100px`, background of `gray`, aligned to the right, and presented in white and bold text.
- the calculator should have a width of 700px and the display mode should be `flex`
- the ButtonPanel component should have a row with height of 100px
- the Button component should be 25% of the container it's rendered in except for the 0 button which should be 50% of the it's parent container, each button should have centered text, black in colour, default color should be orange, each button should have color and wide as incoming props. If the given button is wide, it should take up 50% of the space else 25%. You need to perform a check like `wide === true ? 50% : 25%`
- each Button should have borders

Reply:
Implemented all of them. Thanks for the suggestions :)